### ISSUE TITLE
Decreasing NYT10 Performance at every training epoch

### ISSUE THREAD

**Issue by pvcastro** [slczgwh/REDN#12]
Hi there!

I'm training on the NYT-10 dataset using the default hyperparameters from the repository, without changing anything but cuda_device, setting num_workers to 0 and batch_size to fit in my GTX-1070.

Training starts with a good accuracy/f1 in the first epoch, but decreases at every following epoch. Should I change anything else so the training can work on this dataset?

Thanks!

**Comment by nannigath** [comment]
@pvcastro  were you able to fix it? what was the final f1 score?

**Comment by slczgwh** [comment]
Good question. We also find this problem.  It is because NYT10 are evaluated with no Negative samples(NA), while training with around 80% negative samples. Thus the model will be overfitted on negative samples(if you add NA in evaluating, you will find the scores increase). It is better to stop early.

**Comment by slczgwh** [comment]
It also reminds us we'd better to keep similar statistic distribution between training set and testing set while using in a real situation.

**Comment by pvcastro** [comment]
> @pvcastro were you able to fix it? what was the final f1 score?

Sorry, missed here. No, I haven't tried running this training again.
